GameStudies.js

```diff
@@ -0,0 +1,108 @@
+{
+	"translatorID": "8c2c716c-09ba-421a-ae23-a1590d90a95e",
+	"label": "Game Studies",
+	"creator": "John Weis",
+	"target": "gamestudies\\.org",
```

Comment:
could we be more restrictive here? ```^https?://gamestudies\\.org/``` (ending slash is also prefered since it works bettet with proxies)